docs/answers.md

```diff
@@ -38,6 +38,58 @@ const annotateData = (array) => array.reduce(reducer, [])
 const newData = annotateData(data)
 ```
 
+## Objects and Classes
+
+### Delays
+
+```{js}
+class Delay {
+  constructor(initialValue){
+    this.nextValue = initialValue
+  }
+
+  call(nextValue) {
+    let previousValue = this.nextValue
+    this.nextValue = nextValue
+    return previousValue
+  }
+}
+```
+
+### Filtering
+
+```{js}
+class Filter {
+  constructor(...values){
+    this.filterValues = values
+  }
+
+  call(inputValue){
+    return this.filterValues.some((value) => value === inputValue) ? null : inputValue
+  }
+}
+```
+
+### Pipelines
+
+```{js}
+class Pipeline {
+    constructor(...pipes){
+        this.pipes = pipes
+    }
+
+    call(inputValue){
+        let returnValue
```

Comment:
There is no need to introduce another variable: you can update `inputValue` repeatedly with `inputValue = pipe.call(inputValue)`. (Although if you're going to do this, it should probably just be called `value`.)